src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -2313,6 +2338,13 @@ bool QgsPostgresProvider::addFeatures( QgsFeatureList &flist, Flags flags )
         continue;
 
       QString fieldname = mAttributeFields.at( idx ).name();
+
+      if ( mGeneratedValues.contains( idx ) && !mGeneratedValues.value( idx, QString() ).isEmpty() )
```

Comment:
```suggestion
      if ( !mGeneratedValues.value( idx ).isEmpty() )
```

That should be sufficient

Reply:
Sure, resolved with commit 793409c.